docs/style-guide-for-curriculum-challenges.md

```diff
@@ -201,3 +201,128 @@ We want campers to achieve a flow state. We want them to build momentum and blas
 Creating these challenges requires immense creativity and attention to detail. But you'll have plenty of help. You have support from a whole team of contributors, whom you can bounce ideas off of and demo your challenges to. Stay active in the [contributors room](https://gitter.im/freecodecamp/contributors) and ask lots of questions.
 
 With your help, we can design an interactive coding curriculum that will help millions of people learn to code for years to come.
+
+## Reply Templates for Moderators Reviewing Pull Requests
+
+> You can make your own with GitHub's built in [**Saved replies**](https://github.com/settings/replies/) feature or use the ones below.
+
+### Thank you
+
+```markdown
+Thank you for your contribution to the page! 👍
+We're happy to accept these changes, and look forward to future contributions. 🎉
+```
+
+### Thank you and congrats
+
+> For thanking and encouraging first-time contributors.
+
+```markdown
+Hi @username. Congrats on your first pull request (PR)! 🎉
+
+Thank you for your contribution to the page! 👍
+We're happy to accept these changes, and look forward to future contributions. 📝
+```
+
+### Build Error
+
+```markdown
+Hey @username
+
+So I'd love to be able to merge your changes but it looks like there is an error with the Travis CI build. ⚠️
+
+Once you resolve these issues, I will be able to review your PR and merge it. 😊
+
+---
+
+> Feel free to reference the [Style guide for writing articles](https://github.com/freeCodeCamp/freeCodeCamp#article-title) for this repo on formatting an article correctly so your Travis CI build passes. ✅
+>
+> Also, it's good practice on GitHub to write a brief description of your changes when creating a PR. 📝
+```
+
+### Syncing Fork
+
+> When PR is not up to date with `master` branch.
+
+``````markdown
+Hey @username
+
+So I'd love to be able to merge your changes but it looks like there is an error with the Travis CI build. ⚠️
+
+```bash
+Error: ENOTDIR: not a directory, open 'src/pages/java/data-abstraction/index.md'
+```
+
+This particular error was not actually caused by your file but was an old error caused by merging faulty code to the `master` branch. It has since been resolved.
```

Comment:
```suggestion
This particular error was not caused by your file but was an old error caused by merging faulty code to the `master` branch. It has since been resolved.
```